packages/react-link/config/tests.js

```diff
@@ -1,14 +1,7 @@
 /** Jest test setup file. */
 
 const { configure } = require('enzyme');
-const { initializeIcons } = require('@fluentui/font-icons-mdl2');
 const Adapter = require('enzyme-adapter-react-16');
-const { resetIds } = require('@fluentui/utilities');
-
-// Initialize icons.
-initializeIcons('');
 
 // Configure enzyme.
 configure({ adapter: new Adapter() });
-
-resetIds();
```

Comment:
`react-link` is not using either font icons or `useId()`, this is a safe cleanup